src/components/data/orgs.js

```diff
@@ -0,0 +1,162 @@
+{ name: 'civictechfredericton', id: 33107366 },
+{ name: 'civictechyyc', id: 36767149 },
+{ name: 'opendatabc', id: 1002534 },
+{ name: 'codeforjapan', id: 5137738 },
+{ name: 'codefortokyo', id: 10110858 },
+{ name: 'codeforsapporo', id: 5637003 },
+{ name: 'codeforhakodate', id: 8735805 },
+{ name: 'muroran', id: 10561219 },
+{ name: 'howml', id: 11631697 },
+{ name: 'codefortoda', id: 45053721 },
+{ name: 'codeforichikawa', id: 22729568 },
+{ name: 'codeforchiba', id: 9192544 },
+{ name: 'codeforsetagaya', id: 6940927 },
+{ name: 'codefortoshima', id: 17681119 },
+{ name: 'code4nerima', id: 35978674 },
+{ name: 'codeforchofu', id: 7628029 },
+{ name: 'code48', id: 8231753 },
+{ name: 'openkawasaki', id: 7857102 },
+{ name: 'codeforyokosuka', id: 23724204 },
+{ name: 'codeforniigata', id: 14312166 },
+{ name: 'codeforkanazawa-org', id: 5687728 },
+{ name: 'code4sabae', id: 40041955 },
+{ name: 'code-for-gifu', id: 34079961 },
+{ name: 'code4aichi', id: 36985174 },
+{ name: 'code4nagoya', id: 10399396 },
+{ name: 'open-osaka(notsure)', id: 24203651 },
+{ name: 'codeforamagasaki', id: 22439505 },
+{ name: 'codeforkobe', id: 10889832 },
+{ name: 'code4nara', id: 18328349 },
+{ name: 'code4ikoma', id: 11582910 },
+{ name: 'code4hiroshima', id: 13003502 },
+{ name: 'codeforokinawa', id: 7360004 },
+{ name: 'codeforjapan', id: 5137738 },
+{ name: 'codeforyouth', id: 32522544 },
+{ name: 'codeforafrica', id: 2786364 },
+{ name: 'code4salone', id: 15065220 },
+{ name: 'codeforafrica', id: 2786364 },
+{ name: 'codeforaustralia', id: 2573081 },
+{ name: 'codeforgermany', id: 6952379 },
+{ name: 'codeforberlin', id: 17669902 },
+{ name: 'oklabbonn', id: 11390108 },
+{ name: 'codeforchemnitz', id: 7165089 },
+{ name: 'codeforgiessen', id: 8227138 },
+{ name: 'codeforhamburg', id: 7387840 },
+{ name: 'codeforkarlsruhe', id: 10452685 },
+{ name: 'codeforleipzig', id: 7286986 },
+{ name: 'code-for-magdeburg', id: 10670382 },
+{ name: 'codeformuenster', id: 6464842 },
+{ name: 'codeforniederrhein', id: 16598573 },
+{ name: 'codeforosnabrueck', id: 30509575 },
+{ name: 'open-data-potsdam', id: 19558368 },
+{ name: 'codeforruhrgebiet', id: 17032365 },
+{ name: 'opendata-stuttgart', id: 11635590 },
+{ name: 'ulmapi', id: 985867 },
+{ name: 'opendatal', id: 7932972 },
+{ name: 'epforgpl', id: 3350223 },
+{ name: 'codeforpakistan', id: 4421892 },
+{ name: 'code4romania', id: 18010308 },
+{ name: 'openupsa', id: 26482663 },
+{ name: 'slashroots', id: 693298 },
+{ name: 'codefornl', id: 16212776 },
+{ name: 'codeandomexico', id: 2940113 },
+{ name: 'g0v', id: 2668086 },
+{ name: 'ciudadanointeligente', id: 631035 },
+{ name: 'mysociety', id: 141338 },
+{ name: 'civictechsweden', id: 44578964 },
+{ name: 'opendatakosovo', id: 7871676 },
+{ name: 'sinar', id: 1127396 },
+{ name: 'waagsociety', id: 895755 },
+{ name: 'openknowledgebe', id: 1188392 },
+{ name: 'code4moldova', id: 52112511 }
+]
+
+export let names = new Map();
+
+for(const org of orgs) {
```

Comment:
You can specify this dictionary as such.

{ "code4moldova": 52112511 }

or 

{"code4moldova": { id: 52112511 }}

Keep your retrieval strategy in mind. Are you fetching them by name or by ID?

Reply:
fetching by name in O(1) because when I make the API call on the front end, the only unique identifier is the name of the org, so I parse the names returned from the API call by their GitHub URL string (if there is one) then I look up the parsed name in the map to inject the returned id value in the <img href={https://avatars1.githubusercontent.com/u/${id}?s=100&v=4}/>. sorry if that explanation offended you, just wanted to walk you through my thought process.